packages/ui/src/composites/wui-button/styles.ts

```diff
@@ -114,42 +114,58 @@ export default css`
   }
 
   /* -- Hover & Active states ----------------------------------------------------------- */
-  button[data-variant='main']:hover:enabled {
-    background-color: var(--wui-color-accent-090);
-  }
-
-  button[data-variant='main']:active:enabled {
-    background-color: var(--wui-color-accent-080);
-  }
-
-  button[data-variant='inverse']:hover:enabled {
-    background-color: var(--wui-color-inverse-100);
-  }
-
-  button[data-variant='accent']:hover:enabled {
-    background-color: var(--wui-color-accent-glass-010);
-  }
-
-  button[data-variant='accent-error']:hover:enabled {
-    background: var(--wui-color-error-glass-015);
-    color: var(--wui-color-error-100);
-  }
-
-  button[data-variant='accent-success']:hover:enabled {
-    background: var(--wui-color-success-glass-015);
-    color: var(--wui-color-success-100);
-  }
-
-  button[data-variant='neutral']:hover:enabled {
-    background: var(--wui-color-gray-glass-005);
-  }
-
-  button[data-size='lg'][data-icon-left='true'][data-icon-right='false'] {
-    padding-left: var(--wui-spacing-m);
-  }
-
-  button[data-size='lg'][data-icon-right='true'][data-icon-left='false'] {
-    padding-right: var(--wui-spacing-m);
+  @media (hover: hover) and (pointer: fine) {
+    button[data-variant='main']:hover:enabled {
+      background-color: var(--wui-color-accent-090);
+    }
+
+    button[data-variant='main']:active:enabled {
+      background-color: var(--wui-color-accent-080);
+    }
+
+    button[data-variant='accent']:hover:enabled {
+      background-color: var(--wui-color-accent-glass-015);
+    }
+
+    button[data-variant='accent']:active:enabled {
+      background-color: var(--wui-color-accent-glass-020);
+    }
+
+    button[data-variant='accent-error']:hover:enabled {
+      background: var(--wui-color-error-glass-020);
+      color: var(--wui-color-error-100);
+    }
+
+    button[data-variant='accent-error']:active:enabled {
+      background: var(--wui-color-error-glass-030);
+      color: var(--wui-color-error-100);
+    }
+
+    button[data-variant='accent-success']:hover:enabled {
+      background: var(--wui-color-success-glass-020);
+      color: var(--wui-color-success-100);
+    }
+
+    button[data-variant='accent-success']:active:enabled {
+      background: var(--wui-color-success-glass-030);
+      color: var(--wui-color-success-100);
+    }
+
+    button[data-variant='neutral']:hover:enabled {
+      background: var(--wui-color-gray-glass-005);
+    }
+
+    button[data-variant='neutral']:active:enabled {
+      background: var(--wui-color-gray-glass-010);
+    }
```

Comment:
@svenvoskamp I think for we need to lower the colors for hover-active states. Instead the hover should have `002` and active should have `005`, as I'm trying to confirm in designs: https://www.figma.com/design/pJE9nilzQmGufGsOy3HciG/Product-Library?node-id=1-4064&m=dev

<img width="1145" alt="Screenshot 2024-07-23 at 11 19 13" src="https://github.com/user-attachments/assets/185444f1-09ae-4749-b7ea-8da218eabaa6">

The active state's bg looks have too much light;

https://github.com/user-attachments/assets/a8ef70be-26f8-4d27-861d-c49cdc61e0e1

Reply:
Updated button neutral active and hover state to correct ones. All hover/active are now according to design. We can evaluate later with design team to make changes.